src/creator.html

```diff
@@ -677,13 +678,27 @@ <h2>
 													ng-repeat="match in answer.matches"
 													ng-click="removeAnswerMatch($index, $parent.$index)">{{match}}</button>
 											</div>
-											<div class="match-options">
-												<input type="checkbox"
-													ng-model="answer.caseSensitive" id="caseSensitive"/>
-												<label for="caseSensitive">Make Matches Case Sensitive</label>
-												<input type="checkbox"
-													ng-model="answer.characterSensitive" id="characterSensitive"/>
-												<label for="characterSensitive">Make Matches Special Character Sensitive ('$', '#', '%', etc.)</label>
+											<div class="match-options-container">
+												<h4>Matching Options</h4>
+												<p>The response entered by the user will be compared to each possible answer provided above. By default, matches are not case sensitive and whitespace is removed.</p>
+												<ul class="match-options">
+													<li>
+														<input type="checkbox"
+															ng-model="answer.caseSensitive" id="caseSensitive"/>
+														<label for="caseSensitive">Make matches case sensitive</label>
```

Comment:
Labels aren't being attached to their inputs due to multiple answers sharing the same ID. Using the index should work, since no styles are attached to the IDs:
```
<input type="checkbox" ng-model="answer.caseSensitive" id="caseSensitive-{{$index}}"/>
<label for="caseSensitive-{{$index}}">Make matches case sensitive</label>
```

Reply:
Good catch!